[BUG] `Failed to open SQLite database at: /data/meshmonitor.db`

**Describe the bug**

When following https://github.com/Yeraze/meshmonitor/blob/main/docs%2Fdeployment%2FPROXMOX_LXC_GUIDE.md after running `systemctl start meshmonitor`, `systemctl status meshmonitor` reveals the process has exited and not started.

I receive the following error in `journalctl -u meshmonitor`:

```
May 12 05:11:17 meshmonitor meshmonitor[884]: [ERROR] ❌ DATABASE OPEN ERROR ❌
May 12 05:11:17 meshmonitor meshmonitor[884]: [ERROR] ═══════════════════════════════════════════════════════════
May 12 05:11:17 meshmonitor meshmonitor[884]: [ERROR] Failed to open SQLite database at: /data/meshmonitor.db
May 12 05:11:17 meshmonitor meshmonitor[884]: [ERROR]
May 12 05:11:17 meshmonitor meshmonitor[884]: [ERROR] Error: The module '/opt/meshmonitor/node_modules/better-sqlite3/build/Release/better_sqlite3.node' was compiled against a different Node.js version using NODE_MODULE_VERSION 127. This version of Node.js requires NODE_MODULE_VERSION 137. Please try re-compiling or re-installing the module (for instance, using `npm rebuild` or `npm install`).
```


**To Reproduce**
Steps to reproduce the behavior:
1. Follow https://github.com/Yeraze/meshmonitor/blob/main/docs%2Fdeployment%2FPROXMOX_LXC_GUIDE.md

**Expected behavior**
For meshmonitor to start and not exit.

**Environment (please complete the following information):**
* Host OS: Proxmox Virtual Environment 9.1.9
* Deployment type: LXC

--- a/.github/workflows/lxc-template-build.yml
+++ b/.github/workflows/lxc-template-build.yml
@@ -29,7 +29,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v6
         with:
-          node-version: '22'
+          node-version: '24'
           cache: 'npm'
 
       - name: Install dependencies

--- a/lxc/build-lxc-template.sh
+++ b/lxc/build-lxc-template.sh
@@ -186,6 +186,9 @@ cp -r "$PROJECT_ROOT/protobufs" "$ROOTFS_DIR/opt/meshmonitor/"
 cp "$PROJECT_ROOT/package.json" "$ROOTFS_DIR/opt/meshmonitor/"
 cp "$PROJECT_ROOT/package-lock.json" "$ROOTFS_DIR/opt/meshmonitor/"
 
+echo "Rebuilding native modules against container Node.js..."
+chroot "$ROOTFS_DIR" bash -c "cd /opt/meshmonitor && npm rebuild better-sqlite3 --build-from-source"
+
 # Copy Docker helper scripts (apprise-api.py and others)
 mkdir -p "$ROOTFS_DIR/opt/meshmonitor/docker"
 cp "$PROJECT_ROOT/docker/apprise-api.py" "$ROOTFS_DIR/opt/meshmonitor/docker/"

